### DISCUSSION TITLE
How to change infinite loop 100 iterations limit?

### DISCUSSION THREAD

**Discussion by qarmin** [astral-sh/ruff#7459] [Q&A]
I wanted to set it to ~1000 or more to hide false positives, that I see in some libraries, but I cannot find such option in cli help.

**Comment by dhruvmanila** [comment]
Hey, do you mean this constant:

https://github.com/astral-sh/ruff/blob/959338d39d684f5888cad9c339242acc1343ab9d/crates/ruff/src/linter.rs#L264-L264

You can't really set this via any CLI option as it's internal to Ruff. May I know how it would help you to hide false positives? I'm assuming this is related to running the fuzzer (?).

**Reply by qarmin** [reply]
Problem I have with this issue - https://github.com/astral-sh/ruff/issues/6754

Some python files from pypi(without any changes), hit this limit
